sdv/sequential/par.py

```diff
@@ -110,6 +110,7 @@ def __init__(
 
         self._sequence_index = self.metadata.sequence_index
         self.context_columns = context_columns or []
+        self._validate_sequence_key_and_context_columns(self._sequence_key, self.context_columns)
```

Comment:
You don't really need to pass these parameters since they're both on self right?

Reply:
Fixed:
https://github.com/sdv-dev/SDV/pull/2108/commits/4c11961d057bf6de3e4b6aca6bd2fe8bcbabe173#diff-f97bec45b00fd2731a7c766f7d860080f74d4355c407317cd4cea7ec7d38274fL113